src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/DetailedVerbosityJsonFormatSearchResultPackage.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq;
+using System.Text.Json.Serialization;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal class DetailedVerbosityJsonFormatSearchResultPackage : NormalVerbosityJsonFormatSearchResultPackage
```

Comment:
This name is very verbose.

Reply:
In the naming of this type we would like to show two properties mainly: verbosity and whether it is exact-match or not.
would `DetailedJsonSearchResultPackage` and `ExactDetailedJsonSearchResultPackage` be better names than the current `DetailedVerbosityJsonFormatSearchResultPackage ` and `ExactMatchDetailedVerbosityJsonFormatSearchResultPackage `?